lib/types/operation.js

```diff
@@ -252,6 +258,36 @@ Operation.prototype.getType = function (param) {
   }
 };
 
+/**
+ * adds an inline schema (model) to a hash, where we can ref it later
+ * @param {object} schema a schema
+ * @return {number} the ID of the schema being added, or null
+ **/
+Operation.prototype.addInlineModel = function (schema) {
+  var len = this.inlineModels.length;
+  var model = this.resolveModel(schema, {});
+  if(model) {
+    this.inlineModels.push(model);
+    return 'Inline Model '+len; // return string ref of the inline model (used with #getInlineModel)
+  }
+  return null; // report errors?
+};
+
+/**
+ * gets the internal ref to an inline model
+ * @param {string} inline_str a string reference to an inline model
+ * @return {Model} the model being referenced. Or null
+ **/
+Operation.prototype.getInlineModel = function(inline_str) {
+  if(/Inline Model/.test(inline_str)) {
```

Comment:
I just saw this.  This regex will match `Inline Model*` but what we really want is `Inline Model {num}`.  So this regex should be `^Inline Model \d+$` so that we only match what we want so we have less of a chance of false positives.